### DISCUSSION TITLE
Separate thresholds for State Timeline panel

### DISCUSSION THREAD

**Discussion by cardinal83** [grafana/grafana#41397] [Dashboards, Panels & Visualizations]
Need the ability to set separate thresholds for different metrics through data transformation in the State Timeline panel. Now only one threshold is being set.

**Comment by DmitrySidorow** [comment]
any info? state timeline is still inferior to the discrete panel in terms of functions

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
